src/CMakeLists.txt

```diff
@@ -0,0 +1,173 @@
+cmake_minimum_required(VERSION 3.22)
+
+include("${CMAKE_SOURCE_DIR}/cmake/CompilerWarnings.cmake")
+
+set(FAKER_SOURCES
+    modules/airline/Airline.cpp
+    modules/animal/Animal.cpp
+    modules/book/Book.cpp
+    modules/color/Color.cpp
+    modules/commerce/Commerce.cpp
+    modules/company/Company.cpp
+    modules/computer/Computer.cpp
+    modules/crypto/Crypto.cpp
+    modules/database/Database.cpp
+    modules/datatype/Datatype.cpp
+    modules/date/Date.cpp
+    modules/finance/Finance.cpp
+    modules/food/Food.cpp
+    modules/git/Git.cpp
+    modules/hacker/Hacker.cpp
+    modules/helper/Helper.cpp
+    modules/image/Image.cpp
+    modules/internet/Internet.cpp
+    modules/location/Location.cpp
+    modules/lorem/Lorem.cpp
+    modules/medicine/Medicine.cpp
+    modules/movie/Movie.cpp
+    modules/music/Music.cpp
+    modules/person/Person.cpp
+    modules/phone/Phone.cpp
+    modules/plant/Plant.cpp
+    modules/science/Science.cpp
+    modules/sport/Sport.cpp
+    modules/string/String.cpp
+    modules/system/System.cpp
+    modules/vehicle/Vehicle.cpp
+    modules/videoGame/VideoGame.cpp
+    modules/weather/Weather.cpp
+    modules/word/Word.cpp
+    common/FormatHelper.cpp
+    common/LuhnCheck.cpp
+    common/StringHelper.cpp
+)
+
+set(FAKER_HEADERS
```

Comment:
Added header files too, so any change will force CMake to avoid using cached headers.